srcpkgs/dosbox-staging/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'dosbox-staging'
+pkgname=dosbox-staging
+version=0.77.0
+revision=1
+build_style=meson
+hostmakedepends="pkg-config ccache"
```

Comment:
`ccache` is always available in the chroot, don't install manually.

Reply:
Done.